controllers/kafkacluster_controller.go

```diff
@@ -85,7 +84,7 @@ type KafkaClusterReconciler struct {
 // +kubebuilder:rbac:groups=networking.istio.io,resources=*,verbs=*
 
 func (r *KafkaClusterReconciler) Reconcile(ctx context.Context, request ctrl.Request) (ctrl.Result, error) {
-	log := r.Log.WithValues("Request.Namespace", request.NamespacedName, "Request.Name", request.Name)
+	log := logr.FromContextOrDiscard(ctx).WithValues("Request.Namespace", request.NamespacedName, "Request.Name", request.Name)
```

Comment:
This can be avoided https://github.com/kubernetes-sigs/controller-runtime/blob/b1d9a1641f312ed8b526506486415a3983d4e6d1/pkg/internal/controller/controller.go#L112 because it adds the same info there.

Reply:
If you check out your logs you can see that 
```
{"level":"info","ts":"2022-02-03T22:12:00.304Z","logger":"controller.KafkaCluster","msg":"Kafka cluster state updated","reconciler group":"kafka.banzaicloud.io","reconciler kind":"KafkaCluster","name":"kafka","namespace":"kafka","Request.Namespace":"kafka/kafka","Request.Name":"kafka","component":"kafka","clusterName":"kafka","clusterNamespace":"kafka"}
```
Namespace and Name already on the log.

Reply:
Thanks for catching it!